src/Training.TruckWorld.Clone.Backend/Training.TruckWorld.Backend/Domain/Entities/User.cs

```diff
@@ -20,6 +20,5 @@ public User(string firstName, string lastName, string emailAddress, UserRole rol
         EmailAddress = emailAddress;
         Role = role;
```

Comment:
etgancha biz roleId ni qilmaydigan qildik shekil

Reply:
Ha hozir to'g'irlab qo'yaman